### ISSUE TITLE
DatePicker: Keyboard focus is not visible on the "Go to Today," "Next month" and "Previous month" buttons

### ISSUE THREAD

**Issue by edsirko** [OfficeDev/office-ui-fabric-react#4619]
### Bug Report

- __Package version(s)__: Repro'd in 5.79.1 and 5.46.0, but NOT in 5.39.1
- __Browser and OS versions__: Chrome, probably all browsers

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (High) 

Products/sites affected: (VSTS)

#### Describe the issue:

When keyboard-navigating the DatePicker, if you focus the "Go to today" button or the up/down arrows to change months (or years), the buttons should be outlined to indicate focus.  This seems to have been a regression between 5.39.1 and 5.46.0.

#### Actual behavior:

They are not outlined to indicate focus.

#### Expected behavior:

The buttons should be outlined to indicate focus.

**Comment by KevinTCoughlin** [comment]
Can confirm repro on Fabric example site. Might be related to the same commit as https://github.com/OfficeDev/office-ui-fabric-react/issues/4620.

Thanks again for the report @edsirko. We'll get this fixed.

**Comment by KevinTCoughlin** [comment]
It looks like this PR fixes the focus styles -- https://github.com/OfficeDev/office-ui-fabric-react/pull/4643

**Comment by KevinTCoughlin** [comment]
Closing as I can confirm there are focus styles in `master`. Thanks for the fix @lorejoh12!
